docs/setup.md

```diff
@@ -0,0 +1,65 @@
+# Setup
+
+This launcher is intended for deployment as a sidecar container alongside an n8n main instance container, ideally both on the same host. The launcher exposes a health check endpoint at `/healthz` on port `5680` for liveness checks, and the n8n main instance does so on port `5681`. The orchestrator (e.g. k8s) can use these to monitor the health of the launcher and of the n8n main instance.
+
+1. Download the **launcher binary** from the [releases page](https://github.com/n8n-io/task-runner-launcher/releases).
+
+2. Create a **config file** on the host at `/etc/n8n-task-runners.json` and make this file accessible to the launcher.
+
+```json
+{
+  "task-runners": [
+    {
+      "runner-type": "javascript",
+      "workdir": "/usr/local/bin",
+      "command": "/usr/local/bin/node",
+      "args": [
+        "/usr/local/lib/node_modules/n8n/node_modules/@n8n/task-runner/dist/start.js"
+      ],
+      "allowed-env": [
+        "PATH",
+        "N8N_RUNNERS_GRANT_TOKEN",
+        "N8N_RUNNERS_N8N_URI",
+        "N8N_RUNNERS_MAX_PAYLOAD",
+        "N8N_RUNNERS_MAX_CONCURRENCY",
+        "NODE_FUNCTION_ALLOW_BUILTIN",
+        "NODE_FUNCTION_ALLOW_EXTERNAL",
+        "NODE_OPTIONS"
+      ]
+    }
+  ]
+}
+```
+
+Task runner config fields:
+
+- `runner-type` is the type of task runner, currently only `javascript` is supported.
+- `workdir` is the path to directory containing the task runner binary.
+- `command` is the command to execute in order to start the task runner.
+- `args` are the args for the command to execute, currently the path to the task runner entrypoint.
+- `allowed-env` are the env vars that the launcher is allowed to pass to the task runner.
+
+3. Set the **environment variables** for the launcher.
+
+- It is required to specify an auth token by setting `N8N_RUNNERS_AUTH_TOKEN` to a secret. The launcher will use this secret to authenticate with the n8n main instance. You will also pass this `N8N_RUNNERS_AUTH_TOKEN` to the n8n main instance as well.
```

Comment:
I would talk about `n8n instance` since it can be either `main` or `worker`

```suggestion
- It is required to specify an auth token by setting `N8N_RUNNERS_AUTH_TOKEN` to a secret. The launcher will use this secret to authenticate with the n8n instance. You will also pass this `N8N_RUNNERS_AUTH_TOKEN` to the n8n instance as well.
```